backend/app/routers/auth.py

```
@@ -4,16 +1,40 @@
+from fastapi import (
+    FastAPI,
+    Depends,
+    HTTPException,
+    Header,
```

Ruff (0.8.2)

`fastapi.Header` imported but unused

Remove unused import

(F401)